### ISSUE TITLE
Webpack: isomorphic-unfetch breaks node build due to wrong esm interop.

### ISSUE THREAD

**Issue by ealmansi** [developit/unfetch#122]
### Bug Report

Requiring `isomorphic-unfetch` in a project built with Webpack targeting node leads to broken code. The issue is due to wrong interoperation between cjs & esm by `isomorphic-unfetch`.

###  How to Reproduce

```
$ mkdir bug-repro && cd bug-repro
$ yarn init -y
$ yarn add isomorphic-unfetch webpack webpack-cli
$ echo "require('isomorphic-unfetch')('https://goo.gl').then(console.log)" > index.js
$ npx webpack index.js --target node --mode development
$ node dist/main.js
```

### Expected Result

```
> Response {
  ...
}
```

### Actual Result

```
webpack:///./node_modules/isomorphic-unfetch/index.js?:3
		return __webpack_require__(/*! node-fetch */ "./node_modules/node-fetch/lib/index.mjs")(url.replace(/^\/\//g,'https://'), opts);
		                                                                                       ^

TypeError: __webpack_require__(...) is not a function
```

### Context

In file `packages/isomorphic-unfetch/index.js`, you can find the following code: `require('node-fetch')`. However, the package `node-fetch` exposes the function `fetch` as the default export in an ESM module [1]. The correct way of requiring a default export from CommonJS is `require('node-fetch').default`; see [2].

[1] https://github.com/bitinn/node-fetch/blob/master/src/index.js#L34
[2] https://github.com/bitinn/node-fetch/issues/450#issuecomment-462695245

### Environment

```
→ lsb_release -a
Description:	Ubuntu 18.04.2 LTS
→ node --version
v10.15.3
→ yarn --version
1.15.2
→ npm ls isomorphic-unfetch webpack
bug-repro@1.0.0
├── isomorphic-unfetch@3.0.0 
└── webpack@4.39.2
```

**Comment by ivan-kleshnin** [comment]
I'm getting

```
You must provide the URL of lib/mappings.wasm by calling SourceMapConsumer.initialize({ 'lib/mappings.wasm': ... }) before using SourceMapConsumer
    at readWasm
```

With `nodemon + unfetch + slonik`. Probably related to this issue.

**Comment by karlhorky** [comment]
I had a similar issue to @ivan-kleshnin when using Slonik with Next.js in my project:

https://github.com/karlhorky/next.js-with-slonik

Fixed it in this commit by switching `isomorphic-unfetch` to `cross-fetch`, like @gabiseabra in https://github.com/emcasa/react-starter/pull/12:

https://github.com/karlhorky/next.js-with-slonik/commit/a61f47949aa367020c1b0d2f6d657e63702cb6a4

**Comment by danieltodonnell** [comment]
Thank you for raising this issue. Fauna will look at it on Monday and seek a prompt response.

**Comment by karlhorky** [comment]
Awesome, thanks @danieltodonnell!

**Comment by danieltodonnell** [comment]
I made a mistake and commented in the wrong issue :(

I meant to write this in the issue in faunadb that was affected by this issue.

**Comment by developit** [comment]
Released in [4.2.0](https://github.com/developit/unfetch/releases/tag/4.2.0). Sorry for the super long wait!
